projects/core/src/user/store/effects/user-addresses.effect.spec.ts

```diff
@@ -56,14 +39,14 @@ const mockUserAddress: Address = {
 
 describe('User Addresses effect', () => {
   let userAddressesEffect: fromUserAddressesEffect.UserAddressesEffects;
-  let userOccService: OccUserService;
+  let userAddressConnector: UserAddressConnector;
   let actions$: Observable<any>;
 
   beforeEach(() => {
     TestBed.configureTestingModule({
       providers: [
         fromUserAddressesEffect.UserAddressesEffects,
-        { provide: OccUserService, useClass: MockOccUserService },
+        { provide: UserAddressAdapter, useValue: {} },
```

Comment:
Do we need to provide mock user adapter when we only access the connector?

Reply:
Connector requires adapter, so either we create mock connector, or the real one with mocked adapter. And because we use `spyOn` anyway, I preffer using the real one in this case.